jsk_fetch_robot/jsk_fetch_gazebo_demo/README.md

```diff
@@ -0,0 +1,57 @@
+jsk_fetch_gazebo_demo
+=====================
+
+- This demo is roseus version of [fetch\_gazebo\_demo](https://github.com/fetchrobotics/fetch_gazebo/tree/gazebo9/fetch_gazebo_demo).
+- You can learn how to use roseus and how ROS components are executed. (e.g. pointcloud, navigation, motion planning, ... etc)
+
+## Usage
+- Build this package like [jsk\_fetch\_startup README](https://github.com/jsk-ros-pkg/jsk_robot/tree/master/jsk_fetch_robot#setup-environment)
+```bash
+mkdir -p catkin_ws/src
+cd catkin_ws/src
+git clone https://github.com/jsk-ros-pkg/jsk_robot.git
+rosdep install --from-paths . --ignore-src -y -r
+cd ..
+catkin build jsk_fetch_gazebo_demo
+source devel/setup.bash
+roslaunch jsk_fetch_gazebo_demo demo.launch
```

Comment:
dow we need these lines?
these lines are alson written in `Launch demo` section.

Reply:
Thank you.
I updated.
https://github.com/jsk-ros-pkg/jsk_robot/pull/1213/commits/eb707feeacaddbb5e3692b470c784af7db1f06f5